modules/cnf-provisioning-real-time-and-low-latency-workloads.adoc

```diff
@@ -279,3 +279,82 @@ For more information, see link:https://access.redhat.com/documentation/en-us/ope
 == Scheduling a workload onto a worker with real-time capabilities
 
 Use label selectors that match the nodes attached to the machine config pool that was configured for low latency by the Performance Addon Operator. For more information, see link:https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/[Assigning pods to nodes].
+
+[id="performance-addon-operator-disabling-CPUs-for-power-consumption_{context}"]
+== Reducing power consumption by taking CPUs offline
+
+You can generally anticipate telecommunication workloads. When not all of the CPU resources are required, the Performance Addon Operator allows you take unused CPUs offline to reduce power consumption by manually updating the performance profile.
+
+. Update the performance profile.
+
+.. Review the YAML file by running the following command:
++
+[source,terminal]
+----
+$ cat my-performance-profile.yaml
+----
++
+.Example output
+[source,yaml]
+----
+apiVersion: performance.openshift.io/v2
+kind: PerformanceProfile
+metadata:
+  name: performance
+spec:
+  additionalKernelArgs:
+  - nmi_watchdog=0
+  - audit=0
+  - mce=off
+  - processor.max_cstate=1
+  - intel_idle.max_cstate=0
+  - idle=poll
+  cpu:
+    isolated: "2-23,26-47"
+    reserved: "0,1,24,25,48-59"
+    offlined: “”
```

Comment:
offlined field is not present in 4.9

Reply:
I added an example from 4.9 and updated it with the `offlined` field for 4.11. 
If there are any other changes from 4.9 to 4.11 that should be reflected in the docs, let me know!

Reply:
ok, understood. It's the only change.
Everything seems ok. Thanks

Reply:
@amolnar-rh  may we should specify that "offlined: "" " doesn't actually offline any cpus. 

Also the offlined: Parameters in spec is optional